src/include/btree_inline.h

```diff
@@ -2269,6 +2269,31 @@ __wt_btcur_bounds_early_exit(
     return (0);
 }
 
+/*
+ * __wt_cbt_clear_all_deleted_items_flag --
+ *     Reset the page all deleted items flag if it has any data other than deleted.
+ */
+static inline void
+__wt_cbt_clear_all_deleted_items_flag(
+  WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE *first_upd)
+{
+    if (cbt->upd_value->tw.stop_txn != WT_TXN_MAX) {
+        if (__wt_txn_upd_value_visible_all(session, cbt->upd_value))
+            ++cbt->page_obsolete_deleted_count;
+        /*
+         * If the selected tombstone is not same as the most recent update in the update list
```

Comment:
the same as

Reply:
Fixed.

Reply:
I mean it should be "is not the same as the most"

Reply:
I rewrite the entire sentence to make it more clear.